docs/log_collection.md

```diff
@@ -8,73 +8,11 @@ Send logs to Datadog from your Android applications with [Datadog's `dd-sdk-andr
 * Record real client IP addresses and User-Agents.
 * Optimized network usage with automatic bulk posts.
 
-**Note**: The `dd-sdk-android` library supports all Android versions from API level 19 (KitKat).
-
 ## Setup
 
-1. Add the Gradle dependency by declaring the library as a dependency in your `build.gradle` file:
-
-    ```conf
-    repositories {
-        maven { url "https://dl.bintray.com/datadog/datadog-maven" }
-    }
-
-    dependencies {
-        implementation "com.datadoghq:dd-sdk-android:x.x.x"
-    }
-    ```
-
-2. Initialize the library with your application context, [tracking consent][7], and the [Datadog client token][2] and Application ID generated when you create a new RUM application in the Datadog UI (see [Getting Started with Android RUM Collection][6] for more information). For security reasons, you must use a client token: you cannot use [Datadog API keys][3] to configure the `dd-sdk-android` library as they would be exposed client-side in the Android application APK byte code. For more information about setting up a client token, see the [client token documentation][2]:
-
-    {{< tabs >}}
-    {{% tab "US" %}}
-
-```kotlin
-class SampleApplication : Application() {
-    override fun onCreate() {
-        super.onCreate()
-
-        val config = DatadogConfig.Builder("<CLIENT_TOKEN>", "<ENVIRONMENT_NAME>", "<APPLICATION_ID>")
-                        .build()
-        Datadog.initialize(this, trackingConsent, config)
-    }
-}
-```
-
-    {{% /tab %}}
-    {{% tab "EU" %}}
-
-```kotlin
-class SampleApplication : Application() {
-    override fun onCreate() {
-        super.onCreate()
-
-        val config = DatadogConfig.Builder("<CLIENT_TOKEN>", "<ENVIRONMENT_NAME>", "<APPLICATION_ID>")
-                        .useEUEndpoints()
-                        .build()
-        Datadog.initialize(this, trackingConsent, config)
-    }
-}
-```
-
-    {{% /tab %}}
-    {{< /tabs >}}
-
-     There is also a utility method which gives you the current state of the SDK. You can use this to check if it was properly   initialized or not:
-
-    ```kotlin
-        if(Datadog.isInitialized()){
-          // your code here
-        }
-    ```
-
-     When writing your application, you can enable development logs. All internal messages in the library with a priority equal  or higher than the provided level are then logged to Android's Logcat.
-
-    ```kotlin
-        Datadog.setVerbosity(Log.INFO)
-    ```
-
-3. Configure the Android Logger:
+1. Follow the [instructions](sdk_initialisation.md) to first initialise the SDK.
```

Comment:
Will it work out-of-the-box on our dos website? Or does it require additional PR to https://github.com/DataDog/documentation?

Reply:
That's really a good question but we already have links like that ....so I do not see why it will not work. Maybe @xgouchet has more insights. The reason why I moved it in a separated file is that this way is much more cleaner and we do not have to duplicate that information on each feature file.

Reply:
No those links **do not** work out of the box, they lead to 404 on the documentation website. If there are others, we need to remove them as it will break the user experience. 

I agree that duplication is a bad pattern, but we first need to look at the possible solutions with the documentation team before changing the current file.